apps/syncthing/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  syncthing:
+    image: syncthing/syncthing
+    restart: on-failure
+    container_name: syncthing
+    stop_grace_period: 1m
+    hostname: umbrel-syncthing
+    environment:
+      - PUID=1000
+      - PGID=1000
+    volumes:
+      - ${APP_DATA_DIR}/data:/var/syncthing
+      - ${LND_DATA_DIR}:/var/lnd:ro
+    ports:
+      - 8384:8384
+      - 22000:22000
```

Comment:
Looks like maybe we need to open up port 22000 too?

https://github.com/linuxserver/docker-syncthing/blob/d538b847d3a0035ff0425a9b382d1e555be41440/README.md?plain=1#L89

And maybe also port 21027?

https://github.com/linuxserver/docker-syncthing/blob/d538b847d3a0035ff0425a9b382d1e555be41440/README.md?plain=1#L90

Also looks like the linuxserver image is more popular than the official one. Maybe we should use that instead?